sdk/tables/data-tables/README.md

```diff
@@ -7,49 +7,324 @@ Use the client library to:
 - Create/Delete Tables
 - Query/Create/Read/Update/Delete Entities
 
-[Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/) | [Package (NPM)](https://www.npmjs.com/package/@azure/data-tables) | [API reference documentation](https://aka.ms/js-docs) | [Product documentation](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-overview/) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/d09077b506585ed1c24dab85de16ea3a299ed867/sdk/tables/data-tables/samples)
+Azure Cosmos DB provides a Table API for applications that are written for Azure Table storage and that need premium capabilities like:
 
-## Getting started
+- Turnkey global distribution.
+- Dedicated throughput worldwide.
+- Single-digit millisecond latencies at the 99th percentile.
+- Guaranteed high availability.
+- Automatic secondary indexing.
+- The Azure Tables client library can seamlessly target either Azure table storage or Azure Cosmos DB table service endpoints with no code changes.
 
-### Currently supported environments
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/) | [Package (NPM)](https://www.npmjs.com/package/@azure/data-tables) | [API reference documentation](https://aka.ms/js-docs) | [Product documentation](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-overview/) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/tables/data-tables/samples)
 
-- Node.js version 8.x.x or higher
+## Getting started
 
 ### Prerequisites
 
-TODO: content
+Currently supported environments: Node.js version 8.x.x or higher
```

Comment:
I think you want a statement here about browser support as well?

Reply:
Yeah, good catch